content/shmem_atomic_fetch_add.tex

```diff
@@ -59,14 +61,18 @@
 }
 
 \apinotes{
-    None.
+    As of OpenSHMEM~1.4, the use of \FUNC{shmem\_fadd} has been deprecated
+    and should be expected to be removed in a future specification.
+    Its behavior and call signature is identical to the replacement
+    interface, \FUNC{shmem\_atomic\_fetch\_add}.
 }
 
 \begin{apiexamples}
 
 \apicexample
-	{The following \FUNC{shmem\_fadd} example is for \CorCpp{} programs:}
-        {./example_code/shmem_fadd_example.c}
+	{The following \FUNC{shmem\_atomic\_fetch\_add} example is for
+        \CorCpp{} programs:}
```

Comment:
The example was updated to use C11 generic selection, so the C++ part is no longer true.  Fix is to update ```\CorCpp{}``` to ```\Celev{}```.

@jamesaross will apply the same change to #30.